src/content/en/updates/2019/08/twas-quickstart.md

```diff
@@ -0,0 +1,221 @@
+
+When the app launches, you’ll be given a list of all applications installed on your device by
+`applicationId`.
+Filter the list by the `applicationId` you chose earlier and click on that entry.
+You’ll see a page listing your app’s signature and with a generated Digital Asset Link.
+Click on the Copy or Share buttons at the bottom to export it however you like (eg, save to Google
+Keep, email it to yourself).
+
+Put that JSON in a file called `assetlinks.json` and upload it to your website at
+`.well-known/assetlinks.json` (relative to the root).
+
+### Ensuring your asset link file is accessible {: #ensuring-your-asset-link-file-is-accessible }
+
+Now that you’ve uploaded it, make sure you can access your asset link file on a browser - check that
+`https://yourwebsite.com/.well-known/assetlinks.json` resolves to the file you just uploaded.
+
+Jekyll based websites
+If your website is generated by Jekyll (such as GitHub Pages), you’ll need to add a line of
+configuration so that the `.well-known` directory is included in the output.
+Create a file called `_config.yml` at the root of your site (or add to it if it already exists) and
+enter (more information
+[here](https://help.github.com/en/articles/files-that-start-with-an-underscore-are-missing)):
+
+```yml
+# Folders with dotfiles are ignored by default.
+include: [.well-known]
+```
+
+### Adding more keys {: #adding-more-keys }
+
+A Digital Asset Link file can contain multiple apps that are verified for your website and can allow
+multiple signing keys for your app.
+For example, to add a second key just use the
+[Asset Link Tool](https://play.google.com/store/apps/details?id=dev.conn.assetlinkstool) to
+determine the key and add it as a second entry to the `sha256_cert_fingerprints` field.
+The code in Chrome that parses this JSON is quite strict, so make sure you don’t accidentally add an
+extra comma at the end of the list.
+
+```json
+[{  
+  "relation": ["delegate_permission/common.handle_all_urls"],  
+  "target": {  
+    "namespace": "android_app",  
+    "package_name": "com.appspot.pwa_directory",
+    "sha256_cert_fingerprints": [
+      "FA:2A:03:CB:38:9C:F3:BE:28:E3:CA:7F:DA:2E:FA:4F:4A:96:F3:BC:45:2C:08:A2:16:A1:5D:FD:AB:46:BC:9D",
+      "4F:FF:49:FF:C6:1A:22:E3:BB:6F:E6:E1:E6:5B:40:17:55:C0:A9:F9:02:D9:BF:28:38:0B:AE:A7:46:A0:61:8C"
+
+    ]
+  }  
+}]
+```
+
+## Next Steps {: #next-steps }
+
+Hopefully if you’ve followed this guide, you:
+
+* Have built a Trusted Web Activity that verifies correctly.
+* Understand when your debug keys and your release keys are used.
+* Can determine the signature your TWA is being built with.
+* Know how to create a basic Digital Asset Links file.
+
+If not, please have a look at the Troubleshooting section below or file a GitHub issue against
+[these docs](https://github.com/google/WebFundamentals/issues).
+
+For your next steps, I’d recommend you start off by creating an icon for your app. With that done
```

Comment:
We should point them at the specific section in the TWA docs that do that.

Reply:
I pointed them to the Android docs for that (we don't have any TWA docs for it).